packages/react-vapor/rollup.config.js

```diff
@@ -65,9 +68,11 @@ export default [
                 'hogan.js': ['Template', 'compile'],
                 'react-modal': ['setAppElement'],
                 'react-dnd': ['DragDropContext', 'DropTarget', 'DragSource'],
+                diff2html: ['diffChars', 'diffWordsWithSpace'],
             },
         }),
         tsPlugin(),
+        ...(file === 'dist/react-vapor.min.js' ? [terser()] : []),
```

Comment:
What do you think of
```suggestion
        ...(file.includes('.min.') ? [terser()] : []),
```
In case we decide to add `'dist/react-vapor.esm.min.js'`

Reply:
great :cake: